### ISSUE TITLE
[Accordion menu] aria-expended in nested sub-menu's parent will not be updated

### ISSUE THREAD

**Issue by abdullahsalem** [zurb/foundation-sites#7951]
**Description**
Generally in accordion menu, once the sub-menu is closed, *Foundation* will update that sub-menu's attribute `aria-hidden` and its parent `li`'s attribute `aria-expanded`, thus you obviously notice that the arrows work perfectly. 
In nested sub-menus scenario, the nested sub-menus will be closed and its `aria-hidden` will be updated too, but its parent `li`'s `aria-expanded` will not be updated, thus you could notice that the arrows doesn't work correctly.

**Steps to Reproduce**
1- *Open* any nested sub-menu
2- *Close* its most upper parent
3- *Reopen* that most upper parent

**Expected Behavior**
When you *reopen* the most upper parent (*step 3*), you expected to see sub-menu's parent has the proper arrow (*down arrow*), and its `aria-expanded`' is updated to `false`.

**Actual Behavior**
When you *reopen* the most upper parent (*step 3*), the sub-menu's parent *DOSEN'T* have the proper arrow, and its `aria-expanded` is *NOT* updated to `false`.

![accordion-menu-aria-expanded-issue](https://cloud.githubusercontent.com/assets/177020/12466786/f6eb82de-bfe7-11e5-8cd5-cf0481deea3c.png)

**Test Case**
You can test that scenario on the [docs website](http://foundation.zurb.com/sites/docs/accordion-menu.html#basics)

**Comment by abdullahsalem** [comment]
Thanks Chris!
